### PR TITLE
Updates Purchase method to collect customer information and adds sdk identifiers

### PR DESCRIPTION
**Pull request by cb-haripriyan**
- Updates Purchase method to collect customer information
- Adds sdk identifiers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,8 +112,13 @@ Pass the product and customer identifier to the following function when your cus
 In case this parameter is not passed, then the **customerId** will be the same as the **SubscriptionId** created in Chargebee.
 
 ```ts
-import Chargebee, { Purchase } from '@chargebee/react-native-chargebee';
-
+import Chargebee, { Purchase, Customer } from '@chargebee/react-native-chargebee';
```
Comment (cb-sriramthiagarajan) [inline README.md:115]: Nit: Should `Purchase` and `Customer` be imported as types?

File: src/NativeChargebeeReactNative.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export interface Spec extends TurboModule {
   ): Promise<AuthenticationDetail>;
   retrieveProductIdentifiers(queryParams: Object): Promise<Array<string>>;
   retrieveProducts(productIds: Array<string>): Promise<Product>;
-  purchaseProduct(productId: string, customerId: string): Promise<Purchase>;
+  purchaseProduct(productId: string, customer: Object): Promise<Purchase>;
```
Comment (cb-sriramthiagarajan) [inline src/NativeChargebeeReactNative.ts:18]: Can we use the `Customer` type here instead of `Object`?

**Review by cb-sriramthiagarajan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cb-haripriyan) [reply README.md:115]: I believe this would be dependent on the typescript version used in the app.

File: src/NativeChargebeeReactNative.ts (hunk 2 above)
Comment (cb-sriramthiagarajan) [reply src/NativeChargebeeReactNative.ts:18]: Oops yeah, got it.

**Pull request merged**